src/panels/config/automation/condition/types/ha-automation-condition-state.ts

```diff
@@ -18,6 +26,7 @@ const stateConditionStruct = object({
   attribute: optional(string()),
   state: optional(string()),
   for: optional(union([string(), forDictStruct])),
+  enabled: optional(boolean()),
```

Comment:
For the future, we should make a base `condition` struct like <https://github.com/home-assistant/frontend/blob/dev/src/panels/config/automation/structs.ts#L3>